config/settings.yml

```diff
@@ -903,10 +903,21 @@
 :host_scan:
   :queue_timeout: 20.minutes
 :http_proxy:
-  :host:
-  :password:
-  :port:
-  :user:
+  :default:
+    :host:
+    :password:
+    :port:
+    :user:
+  :azure:
+    :host:
+    :password:
+    :port:
+    :user:
+  :amazon:
```

Comment:
Just realized in another PR, that this is `ec2` or `ec2_network`. See [here](https://github.com/ManageIQ/manageiq-providers-amazon/search?utf8=%E2%9C%93&q=ems_type&type=Code)